L1Trigger/Configuration/python/L1Trigger_EventContent_cff.py

```diff
@@ -93,3 +94,20 @@
         'keep LumiSummary_lumiProducer_*_*')
 )
 
+
+if eras.stage2L1Trigger.isChosen():
+    # stage 2 L1 trigger
+    l1Stage2Digis = [
+        'keep *_gtStage2Digis__*', 
```

Comment:
I did not put the star on purpose, as my understanding from the email thread was that for the GT we wanted only the bits (`GlobalAlgBlk` and `GlobalExtBlk`) which have an empty instance label, and not also all the other collections (muons, jets, taus, etc).